File: tests/shared/domain/random_generator.py
from faker import Faker


class RandomGenerator:
    faker = Faker()

    @classmethod
    def uuid(cls) -> str:
        return str(cls.faker.uuid4(cast_to=str))

    @classmethod
    def url(cls) -> str:
        return cls.faker.url()

    @classmethod
    def title(cls) -> str:
        return cls.faker.sentence()

File: backend/src/test/kotlin/com/dimanupy/backend/graph/RandomGenerator.kt
package com.dimanupy.backend.graph

import com.github.javafaker.Faker
import java.util.UUID

object RandomGenerator {
    private val faker = Faker()

    fun title(): String = faker.book().title()

    fun notionUrl(): String = "https://www.notion.so/${faker.internet().slug()}"

    fun uuid(): String = UUID.randomUUID().toString()
}
